internal/pkg/model/receiver_test.go

```diff
@@ -0,0 +1,36 @@
+package model_test
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/model"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMappedColumns(t *testing.T) {
+	t.Parallel()
+
+	input := model.MappedColumns{
+		model.ColumnID{},
+		model.ColumnDatetime{},
+		model.ColumnIP{},
+		model.ColumnBody{},
+		model.ColumnHeaders{},
+		model.ColumnTemplate{
+			Language:               "jsonnet",
+			UndefinedValueStrategy: "null",
+			Content:                `body.my.key+":"+body.my.value`,
+			DataType:               "STRING",
+		},
+	}
+
+	bytes, err := json.Marshal(&input)
+	assert.NoError(t, err)
+
+	var output model.MappedColumns
+	err = json.Unmarshal(bytes, &output)
+	assert.NoError(t, err)
+
+	assert.Equal(t, input, output)
+}
```

Comment:
Tento test ma jednu nevyhodu, ze nevidis vobec aky JSON to generuje.
Rozdelil by som to na 2 testy, alebo by som to v jednom teste porovnaval oproti statickemu stringu.

Chybaju mi tu 2 testy:
- encode a unknown type
- decode a unknown type

Reply:
Encode unknown type nejde, protože nemůžeš do `model.Columns` vložit nic jiného, než typ, který implementuje `Column`. Zbytek jsem upravil.

Reply:
ok, to by si musel v teste pridat nejaky novy typ, ktory nemas registrovany, netreba